bin/hbase

```diff
@@ -757,6 +757,13 @@ elif [ "$COMMAND" = "hbtop" ] ; then
   HBASE_OPTS="${HBASE_OPTS} ${HBASE_HBTOP_OPTS}"
 else
   CLASS=$COMMAND
+  if [ "$CLASS" = "org.apache.hadoop.hbase.IntegrationTestIngest" ] ; then
```

Comment:
There are way more classes than just this one in hbase-it -- in this package alone there are 17 classes with names that start "IntegrationTest". It would be better to look for all the `main` methods in that module and determine some conditional expression to match them.